### PR TITLE
add confirms to info

### PR DESCRIPTION
**Pull request by antb123**
This is useful for traders and for people using anchors for payments

### CODE REVIEW

**Comment by msfeldstein** [comment]
This seems really specific for bitcoin/eth.  I'm not against adding it, but i wonder if there's a way to do it in a more general sense.  I don't want to have super specific fields for all different types of transaction if possible

**Comment by leighmcculloch** [comment]
The concept of a specific number of confirms is unique to blockchain transactions, but the concept of time before funds are available are common to all payment networks. We could generalize the field to refer to the time until the funds will be available, where the unit of time is customizable and can be seconds, minutes, or blockchain confirmations.

```markdown
* `eta`: Optional. Estimated time until the funds will be sent to the address.
* `eta_units`: Optional. Required if `eta` is specified. Units that `eta` is expressed in. Valid values:
   * `seconds`
   * `confirms`
```

**Comment by msfeldstein** [comment]
I'd be cool with the generalization leigh suggests

I also wonder if this belongs in /info or if the `status_eta` field in `/transaction` response would work (adding `eta_units`).

**Comment by msfeldstein** [comment]
I don't really know what having this field in /info would enable as far as usability

**Comment by leighmcculloch** [comment]
@msfeldstein I think adding it to `/info` would serve to prevent it from being a surprise if the time period is a long period of time. Given that people generally expect blockchain powered payments to be faster than real-world, it's good for an anchor to be upfront if their USD to token deposit flow that uses ACH is going to be 5-7 days turn around.

**Comment by msfeldstein** [comment]
Sgtm, one the pr is updated I'll merge

File: ecosystem/sep-0024.md [hunk 1]
```diff
@@ -443,6 +443,7 @@ The JSON object contains an entry for each asset that the anchor supports for SE
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
+* `confirms`: Optional number of confirms before the crypto transation will be released on stellar.
```
Comment (kiafer35) [inline ecosystem/sep-0024.md]: qp7yukqy94a70xgrwrqy3n9aymmkrmv59ukdqzaktt

Comment (kiafer35) [reply ecosystem/sep-0024.md]: kiafer

File: ecosystem/sep-0024.md [hunk 2]
```diff
@@ -443,6 +443,7 @@ The JSON object contains an entry for each asset that the anchor supports for SE
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
```
Comment (kiafer35) [inline ecosystem/sep-0024.md]: qp7yukqy94a70xgrwrqy3n9aymmkrmv59ukdqzaktt

**Comment by msfeldstein** [comment]
@antb123 if you still want this please update to the suggestion leigh mentioned and reopen, closing for now since its inactive

**Pull request closed without merging**